### PR TITLE
plplot: fix cmake error when building `--with-fortran`

### PR DESCRIPTION
**Pull request by zbeekman**
Reported upstream at https://sourceforge.net/p/plplot/bugs/186/

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by zbeekman** [comment]
Issue reported upstream at https://sourceforge.net/p/plplot/bugs/186/

This fixes #15851

File: Formula/plplot.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@ class Plplot < Formula
   homepage "https://plplot.sourceforge.io"
   url "https://downloads.sourceforge.net/project/plplot/plplot/5.12.0%20Source/plplot-5.12.0.tar.gz"
   sha256 "8dc5da5ef80e4e19993d4c3ef2a84a24cc0e44a5dade83201fca7160a6d352ce"
-  revision 1
```
Comment (zbeekman) [inline Formula/plplot.rb]: I didn't know that we no longer need to add revision lines when patching formulae. Just curious when this changed and if it's documented somewhere.

Comment (ilovezfs) [inline Formula/plplot.rb]: In this case, no revision because the option is non-default. Revision implies forced upgrade.

**Comment by ilovezfs** [comment]
Thanks @zbeekman!

**Comment by zbeekman** [comment]
Ah yes I see. Thanks!

**Pull request merged**